sample_configs/config_supervised_embeddings_duckling.yml

```diff
@@ -0,0 +1,7 @@
+language: "en"
+
+pipeline:
+- name: "intent_featurizer_count_vectors"
+- name: "intent_classifier_tensorflow_embedding"
+- name: "ner_duckling_http"
+  url: "http://duckling:8000"
```

Comment:
Should we use env variables for the duckling url?

Reply:
possibly, but that's for another issue